Makefile

```diff
@@ -0,0 +1,10 @@
+build:
+	nimble build --gc:arc
+
+setup:
+	flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo
+	flatpak install --user org.gnome.Sdk/x86_64/3.38 -y
+	flatpak install --user org.freedesktop.Sdk.Extension.rust-stable/x86_64/19.08 -y
```

Comment:
It doesn't matter to Flathub, but you want `20.08` here.

Reply:
Wow I actually don't even want the rust extension there at all, I copied it over from my other project and it stayed that way. Delted.